### PR TITLE
Image, input components added

### PR DESCRIPTION
**Pull request by muskan-gupta-0803**
Created image and input components

### CODE REVIEW

**Comment by kr-sarvesh** [comment]
Please Improve your responsiveness otherwise looks good to me

File: components/Image/image.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+@import "/docs.css";
+
+.img-round{
+    border-radius: 50%;
+    height: 200px;
+    width:200px;
```
Comment (PrashanthreddyAinala) [inline components/Image/image.css:6]: You can avoid using px values

**Review by PrashanthreddyAinala — COMMENTED** [review]
just that px thing can be avoided rest all the code looks great

File: components/Input/input.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    border:none;
+
+}
+.form-label{
+    font-size: 1.1rem;
+}
+
+.form-success{
+    color:#008000;
+    border:1px solid #008000;
```
Comment (teja538) [inline components/Input/input.css]: you can use variables for colors

File: components/Image/image.html [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+                    A Responsive image is the one that adjusts its height and width according to the screen size. Use class "img-responsive" for this component.
+                </p>
+                <div class="comp-container comp-img-container">
+                    <img class="img-responsive" src="https://picsum.photos/200/200" alt="">
+                </div>
+                <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28200%2C178%2C200%2C1%29&t=seti&wt=none&l=htmlmixed&width=746.25&ds=true&dsyoff=15px&dsblur=68px&wc=true&wa=true&pv=32px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cimg%2520class%253D%2522img-responsive%2522%2520src%253D%2522%2522%2520alt%253D%2522%2522%253E"
+                style="width: 100%; height: 157px; border:0; transform: scale(1); overflow:hidden; margin-bottom: 3rem; margin-top: 1rem;"
+                sandbox="allow-scripts allow-same-origin">
+                </iframe>
```
Comment (teja538) [inline components/Image/image.html:94]: Suggestion: you can define a class for iframe and define corresponding stylings there and can reuse that class,
that helps in responsiveness as well

File: components/Input/input.css [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+.input-text{
+    padding: 0.3rem;
+    margin:0.5rem;
+    font-size: 1rem;
+    border-radius: 2px;
+}
+
+.form-validate span{
+    border:none;
+
```
Comment (teja538) [inline components/Input/input.css]: In css remove unwanted spaces. It's a good practice

**Review by teja538 — COMMENTED** [review]
Everything seems good. Use variables for defining colors.

File: components/Image/image.html (hunk 3 above)
Comment (muskan-gupta-0803) [reply components/Image/image.html:94]: Sure, thanks for the review.

File: components/Input/input.css (hunk 4 above)
Comment (muskan-gupta-0803) [reply components/Input/input.css]: Removed.

File: components/Image/image.css (hunk 1 above)
Comment (muskan-gupta-0803) [reply components/Image/image.css:6]: Okay, Just in this it is used.

File: components/Input/input.css (hunk 2 above)
Comment (muskan-gupta-0803) [reply components/Input/input.css]: changed.

**Pull request merged**